### ISSUE TITLE
Picaso Paint Request

### ISSUE THREAD

**Issue by forecaster-cyber** [microsoft/winget-pkgs#19663]
# Package Requested

- [x] I would like someone else to build the manifest.
- [x] I would like help so I can submit the manifest.
- [x] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

Manifest:

Fill out as much of the manifest metadata you can:

```YAML
PackageIdentifier:   ForecasterCyber.PicasoPaint
PackageVersion:     0.5
PackageLocale:       en-US
Publisher:           Serge Brainin
PackageName:        Picaso Paint
License:             MIT
ShortDescription:    Picaso is an open source painting program that grows everyday! 
Installers: 
 - Architecture:     x64
   InstallerType:    exe
   InstallerUrl:     https://github.com/forecaster-cyber/Picaso_open-source/releases/download/0.5/Picaso.exe
   InstallerSha256:  # SHA256 hash calculated from installer
ManifestType: singleton
ManifestVersion: 1.0.0
```

**Comment by forecaster-cyber** [comment]
If this is good enough for the manifest you can use this.

**Comment by quhxl** [comment]
We cannot add this because it is not an installer.

**Comment by OfficialEsco** [comment]
https://github.com/microsoft/winget-cli/issues/182

**Comment by Trenly** [comment]
Close with reason: Resolved by #68813;
